string article

darlanmendonca opened the pull request.

darlanmendonca committed:
string article

darlanmendonca commented:
@khaosdoctor @OdineiRibeiro give a review pls

odineiramone commented on content/en/string.md:
```diff
@@ -0,0 +1,72 @@
+First, the username are wrong, the write is John Snow. Then, to overwrite the value of variable, 
+
+Primeiro, o username está errado, o correto seria John Snow. Então, para sobrescrever o valor da variável, just what we call the variable name again.
+
```
Why you misturou english and portuguesse nas frases?

darlanmendonca replied:
hahua, good, my bad

odineiramone commented on content/en/string.md:
````diff
@@ -0,0 +1,72 @@
+
+```js
+var phrase = 'You know nothing';
+var username = "John"
````
Have any relevant difference between them `'` and `"`?

darlanmendonca replied:
technically not. but the patterns determine single quotes to be used

khaosdoctor replied:
Hmm, I think we should standardize this...

darlanmendonca committed:
remove portuguese phrase in en/string.md

darlanmendonca committed:
translate js comments in pt/string.md

odineiramone commented:
:+1:

khaosdoctor committed:
Spelling, Grammar and Portuguese mingled

There were some portuguese words mingled in the middle of the text, I removed them

khaosdoctor reviewed (approved):
Made some changes, I think this is good to go
